pkgs/top-level/ocaml-packages.nix

```diff
@@ -1701,7 +1701,9 @@ in let inherit (pkgs) callPackage; in rec
 
   ocamlPackages_5_0 = mkOcamlPackages (callPackage ../development/compilers/ocaml/5.0.nix { });
 
-  ocamlPackages_latest = ocamlPackages_5_0;
+  ocamlPackages_5 = ocamlPackages_5_0;
+
+  ocamlPackages_latest = ocamlPackages_5;
```

Comment:
```suggestion
```
Latest does not seem necessary because default should be latest.
What actually is needed is to bump default to latest.